Day-02/rust/emmaboecker/day02.rs

```diff
@@ -0,0 +1,74 @@
+pub fn main() {
+    let input = include_str!("../../input/day02.txt").lines();
```

Comment:
Bitte nicht in einen arbiträren Ordner wechseln, der nicht zu der Dateistruktur dieses Projekts passt

Reply:
Das ist halt da wo bei mir der input liegt. Wo soll ich sonst den input herbekommen?

Außerdem ist das bei der Lösung von gestern auch so